<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_nombre">
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_apellido">
      <property name="text">
       <string>Apellido:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_edad">
      <property name="text">
       <string>Edad:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_identificacion">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>29</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit_nombre"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_apellido"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_edad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_identificacion"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_agregar">
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_buscar">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_resultadospus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_cerrarSesion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar Sesi√≥n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_eliminar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cerrarSesion</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_cerrarSesion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
